frontend/packages/schema-editor/src/components/TreeView/SchemaItemLabel.tsx

```diff
@@ -134,6 +134,7 @@ export const SchemaItemLabel = ({
           <i className={`fa ${icon}`} />
         </span>{' '}
         <span>{getNameFromPointer(selectedNode)}</span>
+        TEST
```

Comment:
I don't think you meant to commit this. 😇

Reply:
Ups :D

Reply:
Det var en test for å sjekke om dere gjør review grundig nok 😝

Reply:
haha 😂